Library/Formula/logentries.rb

```diff
@@ -20,4 +21,29 @@ def install
     bin.install Dir["#{libexec}/bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
   end
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+        <string>#{opt_bin}/le</string>
+        <string>monitor</string>
+        </array>
+        <key>KeepAlive</key>
+        <true/>
+        <key>RunAtLoad</key>
+        <true/>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    system "if [ \"`#{bin}/le --help || echo $?`\" -ne \"4\" ]; then exit 1; fi"
```

Comment:
Unfortunately the agent needs user configuration before it does something meaningful, the --help/--version check is really the only thing that I can run to check that it works after an install.

I'll update that check and push it up.